app/views/users/index.html.erb

```diff
@@ -10,12 +10,18 @@
     <% @users.each do |user| %>
     <li>
       <% if user.name.present? %>
-        <%= link_to user.name, user_path(user.name) %>
+        <%= link_to user.name, user_path(user) %>
+        <% post_count = user.posts.count %>
+        <% if post_count > 0 %>
+          (<%= post_count %> posts)
+        <% else %>
+          (No posts)
+        <% end %>
       <% else %>
         User name not available
       <% end %>
     </li>
-  <% end %>
+    <% end %>
   </ul>
 </body>
```

Comment:
- [ ] Good job so far 👏👏 Please make sure that that user index page is also displaying user's `photo`
- [ ] Also, please make sure that the index view page matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png) ✨ 
- [ ] Kindly ensure that clicking on each user takes us to the user's show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) ⛏ But I'm unable to navigate to the show page 🐛 Even after manually changing the URL, I am unable to see the user show page. Because of this bug, I'm unable to review user show page , please make sure to fix this so that the next reviewer can review it . Kindly have a look at the following video for better clarity 👇





https://github.com/Kidd254/BlogApp/assets/106119105/950a6009-5e88-4383-be6b-bf28bb08c2f0


### _Note to the next reviewer: Please go through the user's show page. Because of the above bug, I'm unable to review it_